framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -538,7 +520,39 @@ public static String getSqlResult(ResultSet resultSet) throws SQLException {
     }
 	return stringBuffer.toString();
   }
-  
+
+  /**
+   * Collects values of one row from current position of specified result set.
+   *
+   * @param resultSet the source of values
+   * @return list with values of one row from current position of specified resultSet
+   * @throws SQLException if result set is not available.
+   */
+  public static List<Object> getRowValues(ResultSet resultSet) throws SQLException {
+    int columnCount = resultSet.getMetaData().getColumnCount();
+    List<Object> values = new ArrayList<>();
+    for (int i = 1; i <= columnCount; i++) {
+      try {
+        switch (resultSet.getMetaData().getColumnType(i)) {
+          case Types.BINARY:
+          case Types.VARBINARY:
+            byte[] bytes = resultSet.getBytes(i);
+            if (bytes != null) {
+              values.add(new String(bytes, StandardCharsets.UTF_8.name()));
```

Comment:
Don't we need to handle UTF-16 character set and Date objects?

Reply:
This looks good to me and good improvement. Make sure you run all tests locally to confirm.
+1

Reply:
@cchang738, it was checked with pre-commit tests.